ci/cron/src/Main.hs

```diff
@@ -240,7 +240,7 @@ build_docs_folder path versions latest = do
 
 find_commit_for_version :: String -> IO String
 find_commit_for_version version = do
-    release_commits <- lines <$> shell "git log --format=%H origin/master -- LATEST"
+    release_commits <- lines <$> shell "git log --format=%H --branches='*' -- LATEST"
```

Comment:
Applying this to master as well doesn’t do any harm or at least not more harm than only applying it here right? It seems nicer to not have those go needlessly out of sync.

Reply:
This one is the master one.

Reply:
branches are confusing :facepalm: my question still stands: can we apply this to both? if not, maybe worth adding a comment in the code?

Reply:
The Azure cron only runs the latest master commit, so there is little point in changing this code on the 1.0 release branch.

The whole point of release branches being that they get the absolute minimum amount of changes to fix bugs, I'm somewhat reluctant to change anything I don't absolutely have to over there.

Reply:
Oh makes sense, forgot that the cron job will always be based on master.